Apps/Mandelbrot/Mandelbrot.gcl

```diff
@@ -73,7 +73,7 @@ $1ff [do i= A= B= 1<< SquareTable+ ptr= SlowMulShift7! ptr: i 1- if>=0loop]
     
 
 {-----------------------------------------------------------------------+
-|} $0300 \vLR: ret{          RAM page 3                                      |
+|} $0300 \vLR: [ret]{          RAM page 3                                      |
```

Comment:
Writing [ret] instead of ret prevents compilegcl.py from creating a stub in the screen area.